### DISCUSSION TITLE
Issue with TC-Wet, Bright and Disturbance in LSP TSA Analysis on CAT (SPLITS)

### DISCUSSION THREAD

**Discussion by prenner-eo** [davidfrantz/force#154] [Q&A]
Dear all,

I am working on a TSA product (2000-2020) and want to create a CAT Analysis utilising SPLITS over some indices (NDVI, kNDVI, TC-B, TC-W, TC-G, TC-D) in an LSP analysis (Peak of Season). Most data is processed perfectly (NDVI, kNDVI, TC-G), but there are timesteps in the LSP file looking some kind of corrupted, for example, 2018. I am using Landsat 4,5,8 and Sentinel-2A/B data which are processed, resolution merged and co-registrated in the level-2 processing step.

TC-Green 2018:
![image](https://user-images.githubusercontent.com/50298512/146549762-1d86455f-d0d2-414e-8a96-8915224abfab.png)

TC-Bright 2018:
![image](https://user-images.githubusercontent.com/50298512/146549888-596d40a0-d4b1-411a-a58c-227702bd1be1.png)

TC-Wet 2018:
![image](https://user-images.githubusercontent.com/50298512/146549974-1cd9131e-a145-4cb8-8963-0acb188a8f64.png)

TC-Disturbance 2018:
![image](https://user-images.githubusercontent.com/50298512/146550180-500996b4-dbe9-4107-b40c-f866849fd8f9.png)

There is no logical explanation for these fragments as far as I can see; the years before and after look totaly usual (like the TC-Green) and other values are processed good the same year too like the NDVI or TC-Green.

The sensors used for the calculation don't change in FORCE from TC to TC.
![image](https://user-images.githubusercontent.com/50298512/146551178-54152719-28be-4860-9f16-9d33279301bf.png)

Here my PRM-file:
[TSA_2000-2020.txt](https://github.com/davidfrantz/force/files/7735298/TSA_2000-2020.txt)

I changed the interpolation method, changed the indices to singel processing not all in one, another period (2015-2020), another resolution (30m and 10m). The data as I can see looks totaly normal to me, as the other indices are processed normal. I couldn't fix the issue for a long time now and running out of ideas.

I hope someone has another idea to help me out. If you need some more information on this, I would make them available.

Best
Paul

**Comment by davidfrantz** [comment]
Hi Paul,

sorry for the late reply..

We have experienced difficulties with the SPLITS code as well.. In response to this, we have implemented a polar-based LSP approach. Can you please try if these phenometrics work better for you? They are usually much more robust and do not contain that amount of nodata.

Cheers,
David

**Reply by prenner-eo** [reply]
Hey David,

I just tried the polar-based LSP approach, and the data looks much better now. Thank you for your reply.

I would like to make a comment on the documentation in the parameter file.
„Threshold for detecting Start of Season in the cumulative time series.“
(https://force-eo.readthedocs.io/en/latest/components/higher-level/tsa/param.html)

At first, it was not quite clear to me what the values for the threshold refer to. I thought it meant the start, mid and end of season within the year and not that it referred to the phenological year. I only realised this in the paper for the approach (https://www.mdpi.com/1999-4907/11/6/606/htm). Perhaps you could add a sentence or two about what this value refers to in order to make things clearer.

Best regards
Paul
